src-script/pack-apack-mac.js

```diff
@@ -5,31 +5,54 @@ const sevenBin = require('7zip-bin')
 const pathTo7zip = sevenBin.path7za
 
 exports.default = async function (buildResult) {
-  buildResult?.artifactPaths?.forEach((element) => {
+  for (const element of buildResult?.artifactPaths || []) {
     if (
       (element.includes('mac') ||
         element.includes('win') ||
         element.includes('linux')) &&
       element.endsWith('.zip')
     ) {
-      const myStream = Seven.add(
-        element,
-        path.join(buildResult.outDir, '../apack.json'),
-        {
-          $progress: true,
-          $bin: pathTo7zip
-        }
-      )
-      // myStream.on('end', function () {
-      //   console.log(myStream.info)
-      // })
+      try {
```

Comment:
Can we get rid of the try catch? Wouldn't our validation for builds fail if this does not exist?

Reply:
sure will do